src/core/lib/gprpp/memory.h

```diff
@@ -30,6 +30,9 @@
 
 // Add this to a class that want to use Delete(), but has a private or
 // protected destructor.
+// Should not be used in new code.
+// TODO(juanlishen): Remove this macro, and instead comment that the public dtor
+// should not be used directly.
```

Comment:
How did these get into this PR?

Reply:
Thank you for catching this, but I can't find this in my commit locally.  It's weird.

I pull+rebased again and it looks gone now.